src/test/ui/lint/lint-missing-doc.stderr

```diff
@@ -79,8 +79,10 @@ LL | pub enum PubBaz {
 error: missing documentation for a variant
   --> $DIR/lint-missing-doc.rs:119:5
    |
-LL |     PubBazA {
-   |     ^^^^^^^
+LL | /     PubBazA {
```

Comment:
This is a regression.  It can easily be cured by modifying `tcx.hir().opt_span`.  If you want to do it, this can wait a follow-up PR.

Reply:
I fixed it.